docs/estimate.md

```diff
@@ -7,6 +7,10 @@ import TabItem from '@theme/TabItem';
 
 Taquito's estimate method can be used to estimate fees, gas, and storage associated with an operation.
 
+:::note
+Our estimation of `fee`, `gasLimit` and `storageLimit` is based on the response of rpc call to `simulate_operation` plus a small buffer, the context and volume at the time of `simulate_operation` might differ from the context and volume at the time of `injection/operation` which might caused errors like `fees_too_low`, `storage_exhausted.operation` and `gas_exhausted.operation`
```

Comment:
some grammar nitpicks.

Our estimation of 'fee', gasLimit and storageLimit is based on the RPC call response to simulate_operation plus a small buffer. The context and volume during simulate_operation might differ from injection, potentially causing errors such as `fees to low` ......